src/lib.rs

```diff
@@ -510,6 +539,7 @@ impl ModDef {
         self.core.borrow_mut().usage = usage;
     }
 
+    /// Instantiate a module, using the provided instance name. `autoconnect` is an optional list of port names to automatically connect between the parent module and the instantiated module. For example, if the parent module has a port named `clk` and the instantiated module has a port named `clk`, passing `Some(&["clk"])` will automatically connect the two ports. It's OK if some or all of the `autoconnect` names do not exist in the parent module and/or instantiated module; TopStitch will not panic in this case.
```

Comment:
Maybe noteworthy it does not automatically connect instances within this module to each other, it just automatically connects inputs/output from the child to the parent based on name match.

Reply:
Sounds good, I made this clearer in the comment.